### PR TITLE
Cake5

### PR DESCRIPTION
**Pull request by arusinowski**
Initial upgrade to CakePHP 5.0, if accepted please merge this PR into cake5 branch. Thank you

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,9 @@
 language: php
```
Comment (ADmad) [inline .travis.yml:1]: This file can be deleted. We don't use travis anymore.

File: src/Model/Behavior/TagBehavior.php [hunk 2]
```diff
@@ -233,7 +234,9 @@ public function normalizeTags($tags)
                 $result[] = $common + ['id' => $existingTag];
                 continue;
             }
-            list($id, $label) = $this->_normalizeTag($tag);
+            // phpcs:disable SlevomatCodingStandard.Variables.UnusedVariable.UnusedVariable
+            [$id, $label] = $this->_normalizeTag($tag);
```
Comment (ADmad) [inline src/Model/Behavior/TagBehavior.php:238]: Is $id unused? In that case using `[, $label]` should work.

File: src/Model/Entity/TagAwareTrait.php [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
      * Tag entity with given tags.
      *
-     * @param string|array $tags List of tags as an array or a delimited string (comma by default).
+     * @param array|string $tags List of tags as an array or a delimited string (comma by default).
      * @param bool $merge Whether to merge or replace tags. Default true.
-     * @return bool|\Cake\ORM\Entity False on failure, entity on success.
+     * @return \Cake\ORM\Entity|bool False on failure, entity on success.
      */
-    public function tag($tags, $merge = true)
+    public function tag(array|string $tags, bool $merge = true): Entity|bool
```
Comment (ADmad) [inline src/Model/Entity/TagAwareTrait.php:18]: Should we use EntityInterface instead?

File: .travis.yml (hunk 1 above)
Comment (arusinowski) [reply .travis.yml:1]: Done

File: src/Model/Behavior/TagBehavior.php (hunk 2 above)
Comment (arusinowski) [reply src/Model/Behavior/TagBehavior.php:237]: Actually the $label was unused, fixed

**Comment by ADmad** [comment]
Oops sorry, this got auto closed since I deleted the `cake-4.x` branch. Please open a PR again.

**Comment by arusinowski** [comment]
done, PR https://github.com/UseMuffin/Tags/pull/47, thanks

**Pull request closed without merging**